.github/workflows/comment-triggered-ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   scaled_sim:
     uses: ./.github/workflows/run-on-comment.yml
     with:
-      runs-on: self-hosted
+      runs-on: [self-hosted, test]
```

Comment:
We'll need to add this label to all the other self-hosted runners, so that we can better control which workflows run on which machines.